charabia/src/normalizer/arabic.rs

```diff
@@ -202,6 +203,7 @@ mod test {
                     (2, 2),
                 ]),
                 script: Script::Arabic,
+                kind: TokenKind::Word,
```

Comment:
I'm not sure I understand why we need to specify the token kind here now?

Reply:
There are 2 tests, one is a unit test over the current normalizer, and the other is a test running the whole normalization pipeline. In the second case, the classifier is now called and classifies each token forcing us to specify how the token should be classified.

Reply:
I'm feeling there should maybe be a comment in the code specifying that the classification is needed... On the other hand, I'm assuming that if we don't specify this then the test will fail?

If that is the case, we can resolve this.

Reply:
the test fails if this one is missing